### PR TITLE
thieving gloves disguise

### PR DESCRIPTION
**Pull request by Seris02**
## About The Pull Request

disguises thieving gloves from anyone except antags

oh also fixes agent IDs
(probably)

## Why It's Good For The Game

because people examining people and seeing thieving gloves is bad, and I still want them to be recognisable to antags easily

## Changelog
:cl:
add: antag only alternate appearance, adds it to thieving gloves
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
do you want to make it so chameleon (ie agent id) hud actions only show up if you're an antag?

File: code/modules/antagonists/_common/antag_datum.dm [hunk 1]
```diff
@@ -24,6 +24,9 @@ GLOBAL_LIST_EMPTY(antagonists)
 	var/list/blacklisted_quirks = list(/datum/quirk/nonviolent,/datum/quirk/mute) // Quirks that will be removed upon gaining this antag. Pacifist and mute are default.
 	var/threat = 0 // Amount of threat this antag poses, for dynamic mode
 
+/proc/isantagonist(mob/M)
+	return M?.mind?.special_role
+
```
Comment (Ghommie) [inline code/modules/antagonists/_common/antag_datum.dm]: Unneeded proc overhead, remove.

**Review by Ghommie — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/_common/antag_datum.dm (hunk 1 above)
Comment (Seris02) [reply code/modules/antagonists/_common/antag_datum.dm]: weh

**Comment by Detective-Google** [comment]
the gloves should have a description giving away that they're not normal gloves, even if the name is just 'black gloves'

**Pull request closed without merging**